taxi/models.py

```diff
@@ -35,15 +31,12 @@ class Car(models.Model):
         Manufacturer,
         on_delete=models.SET_NULL,
         null=True,
-        related_name="cars"
+        related_name="manufacturers"

```

Comment:
why did you change related_name? the first variant was correct but now you are going to have `manufactur.manufacturers`